lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -1294,6 +1295,23 @@ def run(self):
 
         return msg, changed, rebuilt, differences.get_legacy_docker_diffs(), facts
 
+    def run_safe(self):
+        try:
+            return self.run()
+        except APIError as e:
+            # Sometimes Version.Index will have changed between an inspect and
+            # update. If this is encountered we'll retry the update.
+            if (
+                self.retries > 0
+                and e.explanation
```

Comment:
Once this line doesn't raise an exception, the next one will not raise one, either. So removing this line won't change a thing.
```suggestion
```